common/metrics/config.go

```diff
@@ -233,12 +234,26 @@ func (c *Config) initReportersFromPrometheusConfig(logger log.Logger, customRepo
 	return serverReporter, sdkReporter, nil
 }
 
-func (c *Config) initReporterFromPrometheusConfig(logger log.Logger, config *PrometheusConfig, extensionPoint interface{}) (Reporter, error) {
+func (c *Config) initReporterFromPrometheusConfig(logger log.Logger, config *PrometheusConfig, customReporter interface{}) (Reporter, error) {
```

Comment:
This name is wrong now as it has nothing to do with `PrometheusConfig` in case if `customReporter` is passed.